packages/jest-resolve/src/index.js

```diff
@@ -365,7 +365,7 @@ Please check:
 "moduleNameMapper": {
   "${regex.toString()}": "${chalk.bold(mappedModuleName)}"
 },
-"resolver": ${chalk.bold(resolver)}`),
+"resolver": ${chalk.bold(String(resolver))}`),
```

Comment:
is there a difference between `.toString()` and `String()`?

Reply:
`undefined.toString()` is a type error

Reply:
that's a difference, indeed :D